### PR TITLE
Commit #1. ampere_law

### PR DESCRIPTION
**Pull request by Zulusssss**
Add law and tests.

### CODE REVIEW

File: symplyphysics/laws/electricity/ampere_law.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+## L - length of conductor,
+## B - magnetic induction,
+## a - angle between magnetic induction and current direction.
+
+force = Symbol("force", units.force)
+
+current = Symbol("current", units.current)
+length = Symbol("length", units.length)
+induction = Symbol("induction", units.magnetic_density)
+angle = Symbol("angle", dimensionless)
```
Comment (blackyblack) [inline symplyphysics/laws/electricity/ampere_law.py:29]: You can use `angle_type` here, I think.

File: symplyphysics/laws/electricity/ampere_law.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    validate_output,
+    dimensionless
+)
+
+# Description
+## Ampere's law is the law that determines the force with which a magnetic field acts on
+## a small segment of a conductor with a current. The force turns out to be linearly dependent 
+## on both current and magnetic induction.
+
+## Law is: F = I * L * B * cos(a), where
```
Comment (blackyblack) [inline symplyphysics/laws/electricity/ampere_law.py:17]: Are there any requirements to the conductor geometry. Put them in conditions, if there are any.

File: symplyphysics/laws/electricity/ampere_law.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+def print_law() -> str:
+    return print_expression(law)
+
+
+@validate_input(current_=current,
+    length_=length,
+    angle_=angle,
+    induction_=induction)
+@validate_output(force)
+def calculate_force(current_: Quantity, length_: Quantity, angle_: float, induction_: Quantity) -> Quantity:
```
Comment (blackyblack) [inline symplyphysics/laws/electricity/ampere_law.py:43]: You can accept angle as both Quantity and float, because Quantity allows some angle conversions.

**Review by blackyblack — APPROVED** [review]

File: symplyphysics/laws/electricity/ampere_law.py (hunk 1 above)
Comment (Zulusssss) [reply symplyphysics/laws/electricity/ampere_law.py:29]: > You can use `angle_type` here, I think.

I can't find "angle_type" in the "units" module.

Comment (blackyblack) [reply symplyphysics/laws/electricity/ampere_law.py:29]: It's in `symplyphysics`:
`from symplyphysics import angle_type`

**Review by blackyblack — APPROVED** [review]

**Pull request merged**